tests/addCourse.test.js

```diff
@@ -0,0 +1,76 @@
+const { assert, expect } = require('chai');
+const requests = require('./utils/coursesRequests');
+const { cleanDb } = require('./utils/db');
+
+// Starts the app
+require('../src/app.js');
+
+describe('Add course', () => {
```

Comment:
you should mock the user-service call to prevent the tests fail. You could mock those calls in the mocks.js file. 
Let's talk!

Reply:
something like this

```
const configs = require('../../configs/test');

const usersServiceUrl = url.format(configs.services.usersService.url);

const mockAuth = ({ status = 200, response = {} }) => {
  nock(usersServiceUrl)
    .get('/login')
    .reply(status, response);
};

```

Reply:
Done!